src/core/Akka.Remote/Endpoint.cs

```diff
@@ -716,82 +743,22 @@ private void TryPublish(RemotingLifecycleEvent ev)
     }
 
     /// <summary>
-    /// INTERNAL API.
-    /// 
-    /// Abstract base class for Endpoint writers that require a <see cref="FSM{TS,TD}"/> implementation.
-    /// </summary>
-    internal abstract class EndpointActor<TS, TD> : FSM<TS, TD>
```

Comment:
Deleted the FSM implementation, since we no longer use it anywhere. Refactored away from using FSMs for this stuff a while back.